recipes/splunk-opentelemetry-cpp/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
+
+
+class SplunkOpentelemetryConan(ConanFile):
+    name = "splunk-opentelemetry-cpp"
+    license = "Apache 2.0"
```

Comment:
https://spdx.org/licenses/Apache-2.0.html

```suggestion
    license = "Apache-2.0"
```